src/libs/antares/study/parts/thermal/constant_cost_provider.cpp

```diff
@@ -27,19 +27,19 @@ ConstantCostProvider::ConstantCostProvider(const ThermalCluster& cluster):
 {
 }
 
-double ConstantCostProvider::getOperatingCost(uint serieIndex, uint hourInTheYear) const
+double ConstantCostProvider::getOperatingCost(uint /*serieIndex*/, uint hourInTheYear) const
```

Comment:
[[ maybe_unused ]] is cleaner ? Up to you

Reply:
Yes we can use this attribute